### PR TITLE
add feb product newsletter

### PR DESCRIPTION
**Pull request by glin7**
## What are you changing in this pull request and why?
Upload Feb product email to dev blog

### CODE REVIEW

File: website/blog/2023-02-03-product-update.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
+---
+title: "February 2023 dbt Update"
+description: "The latest product updates from dbt Labs"
+slug: dbt-product-update-2023-february
+# authors: [lauren_craigie]
```
Comment (KiraFuruichi) [inline website/blog/2023-02-03-product-update.md:5]: Can you either add yourself as an author or uncomment Lauren here? Always nice to give a face to blog content.

File: website/blog/2023-02-03-product-update.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
+---
+title: "February 2023 dbt Update"
```
Comment (KiraFuruichi) [inline website/blog/2023-02-03-product-update.md:2]:
```suggestion
title: "February 2023 dbt update"
```

to be consistent with dbt Labs capitalization standards

File: website/blog/2023-02-03-product-update.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+- **[dbt natively supports Python.](https://www.getdbt.com/blog/introducing-support-for-python/)** As of October, you can deploy Python models alongside your SQL ones, enjoying a single view of lineage across both.
+- **[dbt Semantic Layer hit Public Preview](https://docs.getdbt.com/docs/use-dbt-semantic-layer/dbt-semantic-layer)** for Snowflake users in October. Centrally define metrics in your dbt project, and enable anyone to query them from their preferred Semantic Layer integrated tooling.
+
+![https://hs-8698602.f.hubspotemail.net/hub/8698602/hubfs/Screen%20Shot%202023-02-01%20at%209.55.51%20PM.png?upscale=true&width=600&upscale=true&name=Screen%20Shot%202023-02-01%20at%209.55.51%20PM.png](https://hs-8698602.f.hubspotemail.net/hub/8698602/hubfs/Screen%20Shot%202023-02-01%20at%209.55.51%20PM.png?upscale=true&width=600&upscale=true&name=Screen%20Shot%202023-02-01%20at%209.55.51%20PM.png)
+
+## **What's Better?**
```
Comment (KiraFuruichi) [inline website/blog/2023-02-03-product-update.md:28]:
```suggestion
## What's better?
```

File: website/blog/2023-02-03-product-update.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+
+- **[The new dbt Cloud IDE](https://www.getdbt.com/blog/new-improved-cloud-ide/)** is live, and available to everyone. It’s faster, more intuitive, and includes some neat quality-of-life features like git diff view, autocomplete, a code formatter, and an improved syntax highlighting.
+- **[Custom node colors in the dbt DAG](https://www.getdbt.com/blog/announcing-dbt-v1.3-and-utils/).** Color coding your DAG makes it *much* easier to understand your project at a glance. Read about this update, as well as other dbt Core v1.3 improvements.
```
Comment (KiraFuruichi) [inline website/blog/2023-02-03-product-update.md:31]:
```suggestion
- **[Custom node colors in the dbt DAG](https://www.getdbt.com/blog/announcing-dbt-v1.3-and-utils/):** Color coding your DAG makes it *much* easier to understand your project at a glance. Read about this update, as well as other dbt Core v1.3 improvements.
```

File: website/blog/2023-02-03-product-update.md [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+- **[New Service Bundles](https://www.getdbt.com/blog/revamped-professional-services):** The dbt Labs Pro Services team is re-launching its services offerings with new bundles and engagement models, making it simpler for customers to find an engagement scope that works for them.
+- **Removing defaults:** “Run on schedule” is no longer the default setting when creating a new job definition in the dbt Cloud UI. No more accidentally scheduled jobs + unwanted runs.
+
+## **New Resources**
```
Comment (KiraFuruichi) [inline website/blog/2023-02-03-product-update.md:35]:
```suggestion
## New resources
```

File: website/blog/2023-02-03-product-update.md [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+- **[Auto-scaling CI: The intelligent CI](https://docs.getdbt.com/blog/intelligent-slim-ci)** by Doug Guthrie, dbt Labs
+- **[Spin up a dbt+Postgres environment](https://discourse.getdbt.com/t/spin-up-a-dbt-postgres-environment-with-a-single-click-in-vs-code-for-demos-and-interview-assignments/5285/1)** with Panasenco’s template for creating quick demo environments via VS Code.
+- **[Query the dbt Semantic Layer via dbt](https://github.com/Delphi-Data/dbt_natural_language)** with the Delphi API.
+- **[Generate some dogs](https://www.loom.com/share/e33fdfa062dd4764b828002f6cffbf6a)** in the IDE, on the house. 🐶
+
+![https://hs-8698602.f.hubspotemail.net/hub/8698602/hubfs/Screen%20Shot%202023-02-01%20at%208.12.43%20PM.png?upscale=true&width=600&upscale=true&name=Screen%20Shot%202023-02-01%20at%208.12.43%20PM.png](https://hs-8698602.f.hubspotemail.net/hub/8698602/hubfs/Screen%20Shot%202023-02-01%20at%208.12.43%20PM.png?upscale=true&width=600&upscale=true&name=Screen%20Shot%202023-02-01%20at%208.12.43%20PM.png)
```
Comment (KiraFuruichi) [inline website/blog/2023-02-03-product-update.md:46]: This pic renders in low quality; can you potentially upload it as an actual image into the docs repo and use  a lightbox element (that way you can add alt text description to this)? See D[oug's autoscaling CI blog post for an example](https://raw.githubusercontent.com/dbt-labs/docs.getdbt.com/current/website/blog/2023-01-27-autoscaling-ci.md).

File: website/blog/2023-02-03-product-update.md [hunk 7]
```diff
@@ -0,0 +1,76 @@
...
+
+---
+
+## **Weekend Reads**📚
```
Comment (KiraFuruichi) [inline website/blog/2023-02-03-product-update.md:50]:
```suggestion
## Weekend reads📚
```

File: website/blog/2023-02-03-product-update.md [hunk 8]
```diff
@@ -0,0 +1,76 @@
...
+
+*New resources from the community*
+
+- **[ELT eBook for data analysts](https://go.fivetran.com/ebooks/modernize-your-data-transformation-skills-and-accelerate-your-career?utm_medium=partners&utm_source=dbtlabs&utm_campaign=WCT-2022-Q4-Global-FMRR-Modernize-transformaton-skills-and-accelerate-career&utm_content=ebook)**: If you’ve got friends in the data team that aren’t fully up to speed on how the data transformation space has evolved, and how they might get involved, this new ebook from Fivetran and dbt Labs can help!
+- **[The long-awaited Part III of Jonathan Talmi’s blog on dbt at Super](https://medium.com/super/dbt-at-super-part-3-observability-c8755109901f)** covering observability! Don't miss previous chapters on CI and orchestration.
+- **[Sydney Bang's journey to Spotify Analytics Engineer:](https://medium.com/@sydneybang18/how-i-became-an-analytics-engineer-at-spotify-2-years-after-graduating-with-an-exercise-science-e756e9eff6ca)** If you were wondering what a "typical" AE career path looks like... you might be surprised.
+
+## **Places to Go** 🌎
```
Comment (KiraFuruichi) [inline website/blog/2023-02-03-product-update.md:58]:
```suggestion
## Places to go 🌎
```

File: website/blog/2023-02-03-product-update.md [hunk 9]
```diff
@@ -0,0 +1,76 @@
...
+- **[Hex + dbt Labs](https://hex.tech/events/inventa-delivers-operational-analytics/?utm_id=h_7015f000000VLkDAAW&utm_campaign=hexcontent&utm_medium=hex%20event&utm_source=dbt)**: Virtual Feb 15. Learn how [Inventa](https://inventa.shop/) replaced manual workflows with a live Hex app, powered by dbt's Semantic Layer.
+- **[Barcelona:](https://www.meetup.com/barcelona-dbt-meetup/events/291138340/)** In-person Meetup Feb 9, hosted by Spaulding Ridge
+- **[London:](https://www.meetup.com/london-dbt-meetup/events/290846280/)** In-person Meetup Feb 9, hosted by Datatonic
+- **[Auckland:](https://www.getdbt.com/resources/mdsc-countdown/)** In-person event Feb 16, hosted by Countdown
+- **[Chicago:](https://www.meetup.com/chicago-dbt-meetup/events/290180068/)** In-person Meetup Feb 16, hosted by Analytics8
+- **[San Francisco](https://www.getdbt.com/resources/data-dates-sf/?utm_medium=event&utm_source=marketing&utm_campaign=q1-2024_data_dates_event_awareness)**: In-person event Feb 16, hosted by dbt Labs, ThoughtSpot, Fivetran, & InterWorks
+
+## **Consulting Corner** 🌎
```
Comment (KiraFuruichi) [inline website/blog/2023-02-03-product-update.md:70]:
```suggestion
## Consulting corner 🌎
```

File: website/blog/2023-02-03-product-update.md [hunk 10]
```diff
@@ -0,0 +1,76 @@
...
+
+*A few incredibly helpful pieces from our partner network, including many on how to optimize your dbt implementation 🔥*
+
+- **[Querying dbt metrics](https://aimpointdigital.com/querying-dbt-metrics/)** - Aimpoint Digital shows how to use dbt Metrics via the Proxy Server if your BI tool if it isn't yet integrated with the dbt Semantic Layer.
```
Comment (KiraFuruichi) [inline website/blog/2023-02-03-product-update.md:74]:
```suggestion
- **[Querying dbt metrics](https://aimpointdigital.com/querying-dbt-metrics/)**: Aimpoint Digital shows how to use dbt Metrics via the Proxy Server if your BI tool if it isn't yet integrated with the dbt Semantic Layer.
```

File: website/blog/2023-02-03-product-update.md [hunk 11]
```diff
@@ -0,0 +1,76 @@
...
+- **[Simplifying Data Validation with dbt Data Audit](https://medium.com/slalom-build/simplifying-data-validation-with-dbt-ba492d216abe)** Slalom shares how to speed up query validation with the dbt-audit-helper package.
```
Comment (KiraFuruichi) [inline website/blog/2023-02-03-product-update.md:75]:
```suggestion
- **[Simplifying Data Validation with dbt Data Audit](https://medium.com/slalom-build/simplifying-data-validation-with-dbt-ba492d216abe):** Slalom shares how to speed up query validation with the dbt-audit-helper package.
```

File: website/blog/2023-02-03-product-update.md [hunk 12]
```diff
@@ -0,0 +1,76 @@
...
+- **[A Guide to Passing the dbt Analytics Engineering Certification](https://aimpointdigital.com/guide-passing-dbt-analytics-engineering-certification/)**  Join 500+ dbt certified analytics engineers with Aimpoint’s latest guide!
```
Comment (KiraFuruichi) [inline website/blog/2023-02-03-product-update.md:76]:
```suggestion
- **[A Guide to Passing the dbt Analytics Engineering Certification](https://aimpointdigital.com/guide-passing-dbt-analytics-engineering-certification/):**  Join 500+ dbt certified analytics engineers with Aimpoint’s latest guide!
```

File: website/blog/2023-02-03-product-update.md [hunk 13]
```diff
@@ -0,0 +1,76 @@
...
+- **[dbt natively supports Python.](https://www.getdbt.com/blog/introducing-support-for-python/)** As of October, you can deploy Python models alongside your SQL ones, enjoying a single view of lineage across both.
+- **[dbt Semantic Layer hit Public Preview](https://docs.getdbt.com/docs/use-dbt-semantic-layer/dbt-semantic-layer)** for Snowflake users in October. Centrally define metrics in your dbt project, and enable anyone to query them from their preferred Semantic Layer integrated tooling.
+
+![https://hs-8698602.f.hubspotemail.net/hub/8698602/hubfs/Screen%20Shot%202023-02-01%20at%209.55.51%20PM.png?upscale=true&width=600&upscale=true&name=Screen%20Shot%202023-02-01%20at%209.55.51%20PM.png](https://hs-8698602.f.hubspotemail.net/hub/8698602/hubfs/Screen%20Shot%202023-02-01%20at%209.55.51%20PM.png?upscale=true&width=600&upscale=true&name=Screen%20Shot%202023-02-01%20at%209.55.51%20PM.png)
```
Comment (KiraFuruichi) [inline website/blog/2023-02-03-product-update.md:26]: This pic renders in low quality; can you potentially upload it as an actual image into the docs repo and use a lightbox element (that way you can add alt text description to this)? See [Doug's autoscaling CI blog post for an example](https://raw.githubusercontent.com/dbt-labs/docs.getdbt.com/current/website/blog/2023-01-27-autoscaling-ci.md).

**Review by KiraFuruichi — CHANGES_REQUESTED** [review]
Capitalization consistencies, replace images with Lightbox images + actual image uploads to repo

**Comment by runleonarun** [comment]
Closing this as it's maybe a little late to publish. I will check with the PMM team to see if they want to publish their emails updates going forward.

**Pull request closed without merging**